news/_posts/2020-05-11-cms-security-summit.md

```diff
@@ -0,0 +1,89 @@
+# Google CMS Security Summit
+
+Once again, all CMS Security leads shared knowledge together to an event organized by Google in Munich.
+
+
+# Lightingtalks & Security technologies
+
+After spending the evening will stakeholders  getting together and meeting the newcomers over a few beers.
+
+We spent two days at Google Munich office and [Information Security Hub (ISH)](https://www.ish-muc.com/) to speak about what we did since the last years and talk about what's next.
+And of course learnt about each others.
+
+
+We had ligthingtalks as well as conferences about different technologies around the security and data privacy.
+
+As the last year, components like [Strict Content Security Policy](https://bit.ly/strict-csp), [Trusted Types](https://bit.ly/tt-spec) and [Scripting Policy](https://bit.ly/scripting-policy) are again discussed and brought to the forefront.
+
+[Cross-Origin Resource Policy (CORP)](https://fetch.spec.whatwg.org/#cross-origin-resource-policy-header), [Cross-Origin-Embedded-Policy](https://bit.ly/coep-spec) and [Securer Contexts](https://bit.ly/securer-contexts) to enhance isolation and mitigate side-channel attacks.
+
+We also discover the interesting [OpenSK](https://github.com/google/OpenSK) as a [Tock OS](https://www.tockos.org/) application, an open-source security key to be used for multi-factor authentication.
+
+We talked again about the potential of PSR-9 and PSR-10 PHP Standards Recommendations, but since the last year, nothing really moved which is a little sad.
+
+Snyk gave us a talk about [Javascript Prototype Pollution](https://snyk.io/vuln/SNYK-JS-LODASH-450202) which reminds us that we must pay attention about dependencies we used, even if they are famous.
```

Comment:
is Snyk a company or a individual?

Reply:
A company. I add a link on the company name